spec/sidekiq/evss/disability_compensation_form/submit_form0781_spec.rb

```diff
@@ -81,6 +84,257 @@
     end
   end
 
+  describe 'When an ApiProvider is used for uploads' do
+    let(:submission) do
+      Form526Submission.create(user_uuid: user.uuid,
+                               auth_headers_json: auth_headers.to_json,
+                               saved_claim_id: saved_claim.id,
+                               form_json: form0781,
+                               submitted_claim_id: evss_claim_id)
+    end
+    let(:perform_upload) do
+      subject.perform_async(submission.id)
+      described_class.drain
+    end
+    let(:path_to_0781_fixture) { 'spec/fixtures/pdf_fill/21-0781/simple_0781.pdf' }
+    let(:sample_0781_file_read) { File.read(path_to_0781_fixture) }
+    let(:path_to_0781a_fixture) { 'spec/fixtures/pdf_fill/21-0781a/simple.pdf' }
+    let(:sample_0781a_file_read) { File.read(path_to_0781a_fixture) }
+    let(:parsed_0781_form) { JSON.parse(submission.form_to_json(Form526Submission::FORM_0781))['form0781'] }
+    let(:parsed_0781a_form) { JSON.parse(submission.form_to_json(Form526Submission::FORM_0781))['form0781a'] }
```

Comment:
This is just a personal opinion but I'd split the test setup and tests into separate context blocks for 0781 and 0781a to keep this a little more organized